inmap/cmd/config.go

```diff
@@ -119,14 +121,24 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 	}
 
 	for k, v := range config.OutputVariables {
+		v = strings.Replace(v, "\r\n", " ", -1)
```

Comment:
If anything, wouldn't I want to add a third replace function for "\n"? What is your thinking here?